platform/darwin/src/MGLNetworkConfiguration.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MGLNetworkConfiguration : NSObject
+
+/**
+ Returns any custom API base URL in use by instances of MGLMapView and MGLOfflineStorage in the current application.
+ */
++ (nullable NSURL *)apiBaseURL;
```

Comment:
Without a programmatic gett, there’s no point to having a setter, because the developer can just as easily call `[[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMapboxAPIBaseURL"]` themselves.